### PR TITLE
Update cudf dependency to 0.18

### PR DESCRIPTION
**Pull request by NvTimLiu**
Change rapids brannch-0.4 to depend on cuDF v0.18 release jars

Prepare for the for the rapids v0.4.0 release

### CODE REVIEW

**Comment by NvTimLiu** [comment]
build

File: jenkins/Dockerfile-blossom.integration.centos7 [hunk 1]
```diff
@@ -18,7 +18,7 @@
 #
 # Arguments:
 #    CUDA_VER=10.1 or 10.2
-#    CUDF_VER=0.16, 0.17-SNAPSHOT, or 0.18-SNAPSHOT
+#    CUDF_VER=0.16, 0.17-SNAPSHOT, or 0.18
```
Comment (pxLi) [inline jenkins/Dockerfile-blossom.integration.centos7:21]: `0.17-SNAPSHOT` -> '0.17'

Comment (NvTimLiu) [reply jenkins/Dockerfile-blossom.integration.centos7]: fixed

**Comment by NvTimLiu** [comment]
build

**Review by revans2 — APPROVED** [review]
Looks fine to me. Are the 0.18 jars just in the staging repo or in the final repo?

**Comment by GaryShen2008** [comment]
> 
> 
> Looks fine to me. Are the 0.18 jars just in the staging repo or in the final repo?

cuDF 0.18 jar has been released to public maven repo. https://repo1.maven.org/maven2/ai/rapids/cudf/0.18/

**Comment by NvTimLiu** [comment]
> Looks fine to me. Are the 0.18 jars just in the staging repo or in the final repo?

Released in the maven central repo: https://mvnrepository.com/artifact/ai.rapids/cudf/0.18

**Pull request merged**